pkg/os/exec.go

```diff
@@ -16,3 +17,13 @@ func RunWithPrivilege(cmdAndArgs ...string) (string, string, error) {
 	err = cmd.Run()
 	return stdOut.String(), stdErr.String(), err
 }
+
+func RunWithDefaultLocale(command string, args ...string) (string, string, error) {
+	cmd := exec.Command(command, args...)
+	cmd.Env = ReplaceEnv(os.Environ(), "LC_ALL", "C")
+	cmd.Env = ReplaceEnv(cmd.Env, "LANG", "C")
+	stdOut := new(bytes.Buffer)
+	stdErr := new(bytes.Buffer)
```

Comment:
You also need to assign those to `cmd.StdOut` and `cmd.StdErr` 

```golang
cmd.StdOut = stdOut
cmd.StdErr = stdErr
```

Make sure you test thoroughly locally since our CI is not in place. with this PR even you are not able to start.

Reply:
Ahh.. copy paste error from `RunWithPrivilege` :(